source/_integrations/incomfort.markdown

```diff
@@ -21,8 +21,9 @@ ha_config_flow: true
 ---
 
 The `incomfort` {% term integration %} links Home Assistant with your Intergas Lan2RF gateway, including the boiler and any room thermostats attached to it.
+The integration uses the [incomfort](https://pypi.org/project/incomfort-client/) client library.
 
-It uses the [incomfort](https://pypi.org/project/incomfort-client/) client library.
+The Intergas Lan2RF Gateway connects thermostats based on the OpenTherm standard like the [https://www.intergas-verwarming.nl/en/consumer/products/comfort-touch-thermostat/](Comfort Touch Thermostat). The thermostats and LAN2RF gateway are often sold as a set. The gateway is suitable for use with Intergas Kombi Kompakt HRE and HReco appliances from year of manufacture 2014. If the Comfort Touch thermostat together with the gateway, then this will work in combination with Intergas Kombi Kompakt HRE, HReco or Xtreme devices from year of manufacture 2017.
```

Comment:
```suggestion
The Intergas Lan2RF Gateway connects thermostats based on the OpenTherm standard. An example of such a thermostat is the [https://www.intergas-verwarming.nl/en/consumer/products/comfort-touch-thermostat/](Comfort Touch Thermostat). The thermostats and LAN2RF gateway are often sold as a set. The gateway is suitable for use with Intergas Kombi Kompakt HRE and HReco appliances from year of manufacture 2014. If the Comfort Touch thermostat is used together with the gateway, then this will work in combination with Intergas Kombi Kompakt HRE, HReco, or Xtreme devices from year of manufacture 2017.
```
Tiny tweak. Please double-check.